prime-router/src/main/kotlin/cli/SettingCommands.kt

```diff
@@ -519,15 +521,26 @@ class PutMultipleSettings : SettingCommand(
     override val inStream by option("-i", "--input", help = "Input from file", metavar = "<file>")
         .inputStream()
 
+    /**
+     * Number of connection retries.
+     */
+    private val connRetries by option("-r", "--retries", help = "Number of seconds to retry waiting for the API")
```

Comment:
I think this should read "Number of retries" not "Number of seconds to retry", otherwise it's relying on the default value for `pollIntervalSecs`.

Reply:
That was the point.  It is in total number of seconds.  If I was going to expose the setting of interval between retries then maybe